docs/core/preview3/tools/dotnet-clean.md

```diff
@@ -0,0 +1,63 @@
+---
+title: dotnet-clean command | Microsoft Docs
+description: The dotnet-clean command cleans the current directory.
+keywords: dotnet-clean, CLI, CLI command, .NET Core
+author: blackdwarf
+ms.author: mairaw
+ms.date: 10/12/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: eff65fa1-bab4-4421-8260-d0a284b690b2
+---
+
+#dotnet-clean (Tooling )
+
+> [!WARNING]
```

Comment:
not sure what you mean @blackdwarf. adding that line would add the warning similar to the dotnet nuget commands that only exist in csproj: https://review.docs.microsoft.com/en-us/dotnet/articles/core/preview3/tools/dotnet-nuget-delete?branch=pr-en-us-1474